src/config/config.go

```diff
@@ -123,6 +124,15 @@ func (c *Config) ProcessArgs(args []string) {
 		utils.SetNoProxyEnv(c.NoProxy)
 	}
 
+	if highAvailability != nil && *highAvailability != "" {
```

Comment:
I guess once you will use `flagSet.StringVar` you won't need the nil check ? also you will have "" default so yo will not need the `if` at all